composer.json

```diff
@@ -6,7 +6,7 @@
         "magento/framework": "^100.1.0|^101.0.0|^102.0.0|^103.0.0"
     },
     "type": "magento2-module",
-    "version": "1.2.14",
+    "version": "2.0.0",
```

Comment:
Could you also please populate the `CHANGELOG.md` file with the changes made in this version, highlighting any breaking changes that require a major version bump.